exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/DMLTests.kt

```diff
@@ -1460,6 +1461,41 @@ class DMLTests : DatabaseTestsBase() {
             )
         }
     }
+
+    // https://github.com/JetBrains/Exposed/issues/743
+    // need to configure the MySQL server with character_set_server=utf8mb4?
+    // https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-charsets.html
+    @Ignore
```

Comment:
I can't get this test to work. I think we might need to need to configure the server with `character_set_server=utf8mb4`.